00-intro.md

```diff
@@ -64,13 +64,13 @@ Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
 Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
+is more like programming than like using a mouse.
 Commands are terse (often only a couple of characters long),
 their names are frequently cryptic,
 and their output is lines of text rather than something visual like a graph.
 On the other hand,
 the shell allows us to combine existing tools in powerful ways with only a few keystrokes
-and to set up pipelines to handle large volumes of data automatically thus improving productivity and reproducibility.
```

Comment:
I don't know the intention of the author, but in the original sentence he says 

> shell allows us [...] to set up pipelines to handle large volumes of data automatically. 

How would pipes (`|`) help to handle large data automatically? As you said they are good for combining existing tools into pipelines. Should we change the sentence then into:

> [...] with only a few keystrokes shell allows us to combine existing tools into powerful pipelines and handle large volumes of data automatically [...]